client/src/App.jsx

```diff
@@ -8,6 +8,8 @@ import CreateSession from './components/CreateSession.jsx';
 import Join from './components/Join.jsx';
 import GuestMenu from './components/GuestMenu.jsx';
 import HostMenu from './components/HostMenu.jsx';
+import RegisterUser from './components/registerUser.jsx';
```

Comment:
If you change `registerUser` to `RegisterUser` in the file path that should fix the CI error.